documentation/modules/security/proc-installing-certs-per-listener.adoc

```diff
@@ -3,21 +3,41 @@
 // assembly-security.adoc
 
 [id='proc-installing-certs-per-listener-{context}']
-= Providing your own Kafka listener certificates
+= Providing your own Kafka listener certificates for TLS encryption
 
-This procedure shows how to configure a listener to use your own private key and server certificate, called a xref:kafka-listener-certificates-{context}[Kafka listener certificate].
+[role="_abstract"]
+Listeners provide client access to Kafka brokers.
+You configure listeners in the `Kafka` resource, including the configuration required for client access using TLS.
 
-Your client applications should use the CA public key as a trusted certificate in order to verify the identity of the Kafka broker.
+By default, the listeners use certificates signed by the internal CAs generated by Strimzi.
```

Comment:
Since these modules can be standalone, I think you should introduce the expanded wording for CAs first